routes/series.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const router = express.Router()
+const fetch = require('node-fetch')
+require('dotenv').config()
+
+const { API_BASE_URL, API_KEY, API_LANGUAGE } = process.env
+
+router.get('/:id', async (req, res) => {
+  const { id } = req.params
+  const url = `${API_BASE_URL}/${id}?api_key=${API_KEY}&language=${API_LANGUAGE}`
+
+  await fetch(url)
```

Comment:
A nice alternative for `node-fetch` is `axios`. It already does the json parsing for you and has some nice convenience methods.

Reply:
Ok, i will have a look into axios later.